runtime/domain.c

```diff
@@ -1347,9 +1349,12 @@ CAMLprim value caml_ml_domain_interrupt(value domain)
   struct interruptor* target =
     &all_domains[unique_id % Max_domains].interruptor;
 
+  caml_ev_begin("domain/idle_wait");
```

Comment:
This event pair is not the same as the other two since sending interrupt polls for interrupts rather than waiting on a condition variable. Maybe use a different name for this event; perhaps `domain/send_interrupt`.

Reply:
I have changed it to `domain/send_interrupt`. Thanks!